src/Libraries/CoreNodeModels/Properties/Resources.resx

```diff
@@ -206,7 +206,7 @@
     <value>The selected {0}</value>
   </data>
   <data name="FileFromPathDescription" xml:space="preserve">
-    <value>Creates a file object from a path.</value>
+    <value>Creates a file object from a path. A file object loads the selected file in memory for use downstream.</value>
```

Comment:
not sure this addition makes it more clear what the node does? maybe its just me.

Reply:
Maybe its clear enough with current description, ill change it back again to original description.
Addressed in ddbeb47